apps/src/javalab/JavabuilderConnection.js

```diff
@@ -44,10 +44,16 @@ export default class JavabuilderConnection {
     })
       .done(result => this.establishWebsocketConnection(result.token))
       .fail(error => {
-        this.onOutputMessage(
-          'We hit an error connecting to our server. Try again.'
-        );
-        console.error(error.responseText);
+        if (error.status === 403) {
+          this.onOutputMessage(
+            'It appear you are not authorized to access Javalab. Do you need to log in or join a section?'
```

Comment:
appears instead of appear? Or should we just say 'You are currently not authorized...'

Reply:
also should we translate this and the error message below?

Reply:
Ahh thanks, typo. Will ask in slack about preferred message. Was wondering about translation, do we translate other messages in the console?

Reply:
Actually I think you're right, I'll just shorten message :)

Reply:
yes, we translate things like error messages generally